### ISSUE TITLE
github: pare down and eliminate outside collaborators from orgs

### ISSUE THREAD

**Issue by cblecker** [kubernetes/community#3178]
Folks with permissions on the Kubernetes orgs should be members.

We need to work towards getting people who are involved in the org, and remove those collaborators.

I've got two issues for repos, but we need to go through the rest:
https://github.com/kubernetes/minikube/issues/3345
https://github.com/kubernetes/frakti/issues/348

cc: @kubernetes/owners

**Comment by nikhita** [comment]
Just to be clear -- this also means removing folks who are members of the org but have direct collaborator access along with any outside collaborators i.e. people who are _not_ members of the org but have direct collaborator access...right?

**Comment by nikhita** [comment]
We need to the "remove outside collaborators" part into incorporate this into peribolos. xref https://github.com/kubernetes/test-infra/pull/10517.

/assign

**Comment by nikhita** [comment]
Corresponding issue for peribolos: https://github.com/kubernetes/test-infra/issues/10501

**Comment by mrbobbytables** [comment]
Below is an updated list of outside collaborators.

I'll create issues in each repo linking back here / updating this issue with links.  Give folks 30~ days (December 1st)  to respond and/or become members of the associated org. After that they will be removed as an outside collaborator. 

**This does not prevent them from being able to collaborate.**

It just means less permissions in the respective repo.

---

**Key:**
- **[Inactive]** - Defined as no DevStats recorded activity within the past year (< 5 contributions).
- **[Kubernetes org member]** - Member of the Kubernetes Org, but not the org in which the repo resides.


## Kubernetes

- [kubernetes/frakti](https://github.com/kubernetes/frakti)
  - [Crazykev](https://github.com/Crazykev) - Inactive
  - [heartlock](https://github.com/heartlock) - Inactive
  - [bergwolf](https://github.com/bergwolf) - Inactive
  - [ZeroMagic](https://github.com/ZeroMagic)
- [kubernetes/kompose](https://github.com/kubernetes/kompose)
  - [containscafeine](https://github.com/containscafeine) - Inactive
  - [piyush-garg](https://github.com/piyush-garg) - Inactive
  - [pradeepto](https://github.com/pradeepto) - Inactive
  - [surajnarwade](https://github.com/surajnarwade) - Inactive
- [kubernetes/kops](https://github.com/kubernetes/kops)
  - [blakebarnett](https://github.com/blakebarnett)
  - [kenden](https://github.com/kenden) - Inactive
- [kubernetes/kubeadm](https://github.com/kubernetes/kubeadm)
  - [rdodev](https://github.com/rdodev)
- [kubernetes/minikube](https://github.com/kubernetes/minikube)
  - [sanguinecloud](https://github.com/sanguinecloud) - Inactive

## Kubernetes-client

- [kubernetes-client/csharp](https://github.com/kubernetes-client/csharp)
  - [itowlson](https://github.com/itowlson)
  - [tg123](https://github.com/tg123)
- [kubernetes-client/haskell](https://github.com/kubernetes-client/haskell)
  - [guoshimin](https://github.com/guoshimin)
  - [jkachmar](https://github.com/jkachmar)
  - [jonschoning](https://github.com/jonschoning)
- [kubernetes-client/java](https://github.com/kubernetes-client/java)
  - [cliveseldon](https://github.com/cliveseldon)
  - [iocanel](https://github.com/iocanel) - Inactive
  - [kondapally1989](https://github.com/kondapally1989) - Inactive
- [kubernetes-client/javascript](https://github.com/kubernetes-client/javascript)
  - [drubin](https://github.com/drubin) - Kubernetes org member
  - [flyboarder](https://github.com/flyboarder)
  - [itowlson](https://github.com/itowlson)
  - [silasbw](https://github.com/silasbw)
- [kubernetes-client/python](https://github.com/kubernetes-client/python)
  - [jonathan-kosgei](https://github.com/jonathan-kosgei) - Inactive
  - [marun](https://github.com/marun) - Kubernetes org member
  - [mml](https://github.com/mml) - Kubernetes org member
  - [ravisantoshgudimetla](https://github.com/ravisantoshgudimetla) - Kubernetes org member
  - [smarterclayton](https://github.com/smarterclayton) - Kubernetes org member
  - [WilliamDenniss](https://github.com/WilliamDenniss) - Inactive / Kubernetes org member
- [kubernetes-client/ruby](https://github.com/kubernetes-client/ruby)
  - [drubin](https://github.com/drubin) - Kubernetes org member

## Kubernetes-csi

- [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner)
  - [verult](https://github.com/verult) - Kubernetes org member

## Kubernetes-sigs

- [kubernetes-sigs/apiserver-builder-alpha](https://github.com/kubernetes-sigs/apiserver-builder-alpha)
  - [maisem](https://github.com/maisem) - Inactive / Kubernetes org member
  - [metmajer](https://github.com/metmajer) -  Inactive
- [kubernetes-sigs/application](https://github.com/kubernetes-sigs/application)
  - [konryd](https://github.com/konryd) - Inactive / Kubernetes org member
- [kubernetes-sigs/aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator)
  - [mattlandis](https://github.com/mattlandis) - Inactive
  - [mattmoyer](https://github.com/mattmoyer) - Inactive / Kubernetes org member
  - [nckturner](https://github.com/nckturner) - Kubernetes org member
- [kubernetes-sigs/azuredisk-csi-driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver)
  - [ashishranjan738](https://github.com/ashishranjan738)
  - [lizebang](https://github.com/lizebang) - Inactive
  - [Masquerade0097](https://github.com/Masquerade0097) - Inactive
  - [Rui-Tang](https://github.com/Rui-Tang) - Inactive
  - [ZeroMagic](https://github.com/ZeroMagic)
- [kubernetes-sigs/azurefile-csi-driver](https://github.com/kubernetes-sigs/azurefile-csi-driver)
  - [ashishranjan738](https://github.com/ashishranjan738)
  - [lizebang](https://github.com/lizebang) - Inactive
  - [Masquerade0097](https://github.com/Masquerade0097) - Inactive
  - [Pothulapati](https://github.com/Pothulapati) - Kubernetes org member
  - [Rui-Tang](https://github.com/Rui-Tang) - Inactive
  - [ZeroMagic](https://github.com/ZeroMagic)
- [kubernetes-sigs/cluster-api](https://github.com/kubernetes-sigs/cluster-api])
  - [kris-nova](https://github.com/kris-nova) - Kubernetes org member 
- [kubernetes-sigs/cluster-api-provider-aws](https://github.com/kubernetes-sigs/cluster-api-provider-aws)
  - [johanneswuerbach](https://github.com/johanneswuerbach)
- [kubernetes-sigs/kustomize](https://github.com/kubernetes-sigs/kustomize)
  - [barney-s](https://github.com/barney-s)
  - [damienr74](https://github.com/damienr74)
  - [selfmanagingresource](https://github.com/selfmanagingresource) - Inactive
- [kubernetes-sigs/poseidon](https://github.com/kubernetes-sigs/poseidon)
  - [deepak-vij](https://github.com/deepak-vij) - Kubernetes org member
  - [dhilipkumars](https://github.com/dhilipkumars) - Inactive / Kubernetes org member
  - [ICGog](https://github.com/ICGog) - Inactive
  - [ms705](https://github.com/ms705) - Inactive
  - [shivramsrivastava](https://github.com/shivramsrivastava) 

[Kubernetes org member]: https://github.com/kubernetes/org/blob/master/config/kubernetes/org.yaml
[Inactive]: https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All

---

/assign

**Comment by mrbobbytables** [comment]
Almost all outside collaborators have been removed.
-  A few were added to [kubernetes-sigs/kustomize](https://github.com/kubernetes-sigs/kustomize) after the initial notification of clean up. I've updated the issue with the additional people and pinged repo owners on slack. Will continue clean up 1 week from today.
- [kubernetes-sigs/poseidon](https://github.com/kubernetes-sigs/poseidon) - This repo is likely to be archived (https://github.com/kubernetes-sigs/poseidon/issues/178). Archiving will take care of most of the remaining outside collaborators.
